ting.entities.inc

```diff
@@ -65,10 +65,19 @@ class TingEntity extends DingEntity {
       if ($dk5 == 'sk') {
         return '';
       }
+
+      return $dk5;
     }
+
     return FALSE;
   }
 
+  function getClassificationText() {
```

Comment:
> Often DK5 (getClassification()) is placed as a standalone value, whereas DK5-Text (getClassificationText()) is used in conjunction with another value. By experience, this is often with DK5 and/or authors (to extend the shelf placement for the patron).

Thank you for elaborating . That is useful information that could have been provided in DocBlocks.

Sometimes the developers calling these methods might not know the difference and some documentation would go some way to help.